api/proto/flowlog/flowlog.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2017-2019 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+option go_package  = "github.com/lf-edge/eve/api/go/flowlog";
+
+option java_package = "com.zededa.cloud.uservice.proto";
+
+message IpFlow {
+  string src = 1;          // Source address
+  int32  srcPort = 2;      // Source port
+  string dest = 3;         // Destination address
+  string destPort = 4;     // Destination port
+  int32  protocol = 5;     // Protocol
+}
+
+message InstanceInfo {
+  string uuid = 1;           // UUID of the instance
```

Comment:
More clear to say app instance in the comment.

Reply:
It can be the device UUID as well. I can either remove the comment entirely or update it to "UUID of the instance/device". Any preference?

Reply:
Saying "app instance or device UUID" makes sense.
But naming it InstanceInfo is confusing if it can be either.

When would be be for the device?

Reply:
Sure, I can update the comments to "UUID of instance/device". It can be uuid of the device if the flow log is for dom0.
Any suggestion for the name if InstanceInfo is confusing?

Reply:
This message defines the scope for a particular FlowMessage, hence naming it ScopeInfo might make sense.
But one could also inline in the ScopeInfo in the FlowMessage.

As an aside, I find it a lot easier to find things when the top of function starts with the top message as opposed to having to look around for it and find it at the end.

Reply:
OK, I will rename it to 'ScopeInfo'. I will leave it as a separate message instead of inlining it as it makes sense for these to be together logically. We can even use this in other messages in the future.

Reply:
Added the following issue to update api/API.md.
https://github.com/lf-edge/eve/issues/64